nautobot/ipam/api/views.py

```diff
@@ -291,11 +292,18 @@ def available_ips(self, request, pk=None):
 #
 
 
+@extend_schema_view(
+    create=extend_schema(responses={"201": serializers.IPAddressSerializerLegacy}, versions=["1.2"]),
+    list=extend_schema(responses={"200": serializers.IPAddressSerializerLegacy(many=True)}, versions=["1.2"]),
+    partial_update=extend_schema(responses={"200": serializers.IPAddressSerializerLegacy}, versions=["1.2"]),
+    retrieve=extend_schema(responses={"200": serializers.IPAddressSerializerLegacy}, versions=["1.2"]),
+    update=extend_schema(responses={"200": serializers.IPAddressSerializerLegacy}, versions=["1.2"]),
```

Comment:
Do we also need to include `bulk_update`, `bulk_partial_update` as well, i.e. the PUT/PATCH operations on the list endpoint? Or are those handled automatically by the above?